deploy_function.yaml

```diff
@@ -0,0 +1,30 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: ((image_registry))/eq-cloud-function-deploy-image
+    tag: ((deploy_image_version))
+inputs:
+  - name: eq-submission-confirmation-consumer
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_ID:
+  NOTIFY_API_KEY:
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+      gcloud config set project "${PROJECT_ID}"
+
+      cd eq-submission-confirmation-consumer
+      pip3 install pipenv
+      pipenv lock -r > requirements.txt
```

Comment:
I think this could cause an issue. This command will update the lock file which could mean different versions of libraries being deployed on each deploy. Even if we locked versions in the Pipfile, the sub-dependencies wouldn't be locked.

Reply:
Suggestions?

Reply:
What would you suggest we could do?

Reply:
pipenv lock --keep-outdated -d -r > requirements.txt